### ISSUE TITLE
Error with node 0.8.0

### ISSUE THREAD

**Issue by shapeshifta78** [jacobrask/styledocco#32]
Hi,

I installed the newest version and styledocco suddenly stopped working:

```
module.js:340
throw err;
      ^
Error: Cannot find module 'async'
at Function.Module._resolveFilename (module.js:338:15)
at Function.Module._load (module.js:280:25)
at Module.require (module.js:362:17)
at require (module.js:378:17)
at Object.<anonymous> (/usr/local/lib/node_modules/styledocco/cli.js:3:13)
at Module._compile (module.js:449:26)
at Object.Module._extensions..js (module.js:467:10)
at Module.load (module.js:356:32)
at Function.Module._load (module.js:312:12)
at Module.require (module.js:362:17)

exec error: Error: Command failed:
```

The version beforehand that was working like a charm. Am I doing sth. wrong?

**Comment by jacobrask** [comment]
Hm, how did you install it exactly? With npm? If not, don't forget to run npm install in the directory after downloading StyleDocco

**Comment by shapeshifta78** [comment]
Yes, via npm like the version before.

**Comment by shapeshifta78** [comment]
Tried to uninstall and install again, now I get this:

```
/usr/local/lib/node_modules/styledocco/bin/styledocco:40
  options['in'].map(function(file) {
            ^
TypeError: Object ./ has no method 'map'
at Object.<anonymous> (/usr/local/lib/node_modules/styledocco/bin/styledocco:40:17)
at Module._compile (module.js:449:26)
at Object.Module._extensions..js (module.js:467:10)
at Module.load (module.js:356:32)
at Function.Module._load (module.js:312:12)
at Module.runMain (module.js:487:10)
at process.startup.processNextTick.process._tickCallback (node.js:244:9)
```

**Comment by chrisjaure** [comment]
Same here. Version 0.4.0 appears to be broken under node v0.8 as well as v0.6.

**Comment by jacobrask** [comment]
@chrisjaure Is your error also the one about async, or the last one about ./ map?

@shapeshifta78 I fixed the issue with ./ in v0.4.1. Are you still getting the async issue?

Thanks for your reports btw, really appreciated.

**Comment by chrisjaure** [comment]
My error was with .map, but it looks like you've fixed it :)

I think there was another error, but I'll open up a new ticket once I do some investigating.

**Comment by shapeshifta78** [comment]
@jacobrask the asynch one seems to be fixed, but I am still getting the map error like @chrisjaure 
Thanks for fixing the first one.

I am sorry, that I mixed up the issues. Shall I open a new issue for the map-bug?

**Comment by redonkulus** [comment]
Not sure if I should open a new issue for this but on node v0.6.17, I get the following error:

$ styledocco -n "CSS Framework" -o ~/sandbox/style/ assets/ --include assets/base/css/utility.css'
[ '62.5', '%', '/', '1.4' ] 'number' '1.4' 'identifier' 'Helvetica'

/home/y/lib/node_modules/styledocco/node_modules/ncss/lib/parser.js:275
          if (!property || KEYWORDS.indexOf(t) >= 0) { q(t); }
                                    ^
RangeError: Maximum call stack size exceeded

**Comment by chrisjaure** [comment]
@redonkulus That error should be fixed according to #34, though I don't think it's published to npm yet. You'll need to clone the repo it you need the fix immediately.
